content/cpp/concepts/math-functions/terms/nearbyint/nearbyint.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'nearbyint()'
+Description: 'Returns an integer that is closest to a given argument.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`nearbyint()`** function returns an integer that is closest to a given floating point argument.
+
+## Syntax
+
+```pseudo
+nearbyint(n)
+```
+The function takes the argument `n` which represents the positive or any non-negative floating point value. If the last decimal value of `n` is 5 or greater than 5 then the function returns an integer value greater than `n`.
```

Comment:
```suggestion

The function accepts an `n` parameter that represents a positive or negative floating point value. If the tenths decimal value of `n` is 5 or greater, then the function returns an integer value greater than `n`.
```

Reply:
I just tested `nearbyint()` and it seems to work for negative values as well as positive ones.